sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/SessionRecords/TableTests(True)/Table_CreateEnumerableParameterBindingData_CreatesValidParameterBindingDataObjectAsync.json

```diff
@@ -0,0 +1,63 @@
+{
+  "Entries": [
```

Comment:
Not sure what I am missing, but I cannot find test for `CreateEnumerableParameterBindingData_CreatesValidParameterBindingDataObjectAsync`? Can you point me to that?

Reply:
Ah that's an old test that I ended up removing. Let me rebuild the solution and that should fix the json file